torch/nn/modules/activation.py

```diff
@@ -398,6 +398,41 @@ def extra_repr(self) -> str:
         inplace_str = 'inplace=True' if self.inplace else ''
         return inplace_str
 
+class Mish(Module):
+    r"""Applies the Mish function, element-wise.
+    Mish: A Self Regularized Non-Monotonic Neural Activation Function.
+
+    .. math::
+        \text{Mish}(x) = x * \text{Tanh}(\text{Softplus}(x))
+
+    .. note::
+        See `Mish: A Self Regularized Non-Monotonic Neural Activation Function <https://arxiv.org/abs/1908.08681>`.
```

Comment:
Looks like the paper link didn't render correctly 
(see [rendered version](https://13676694-65600975-gh.circle-artifacts.com/0/docs/generated/torch.nn.Mish.html)).

I think you just need an underscore at the end:
```
See `Mish: A Self Regularized Non-Monotonic Neural Activation Function <https://arxiv.org/abs/1908.08681>`_
```

Reply:
Thanks. I will fix it in the next commit. I will commit again after the checks finished in order to make sure that there is no other mistakes.